website/content/configuration/_index.md

```diff
@@ -265,6 +265,24 @@ Addresses can still be specifically allocated from the "expensive"
 pool with the methods described in
 the [usage](/usage/#requesting-specific-ips) section.
 
+### Single-IP address pool
```

Comment:
I think having a dedicated section for this "gotcha" is a bit of an overkill. I suggest we add a `{{% notice note %}}` instead in some place where it makes sense.